packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -647,43 +674,50 @@ export const startRewardDistributor = async ({
       return undefined;
     });
 
-  const feeDistributorFacets = await E(zoe).startInstance(
+  const facets = await E(zoe).startInstance(
```

Comment:
please rename to match https://github.com/Agoric/agoric-sdk/blob/0c6bc9077a852d2a0b0bc6ee1c490274252e2cf3/packages/zoe/src/zoeService/utils.d.ts#L72-L77

```suggestion
  const instanceKit = await E(zoe).startInstance(
```

Reply:
done